k8s/metrics/pyroscope/base/daemonset.yaml

```diff
@@ -31,7 +30,6 @@ spec:
       labels:
         app.kubernetes.io/instance: pyroscope-agent
         app.kubernetes.io/name: pyroscope-agent
-        app.kubernetes.io/version: "0.2.3"
```

Comment:
should be latest???

Reply:
Oh, Thank you for your comment.
It should be the latest :bow: